package.json

```diff
@@ -25,11 +25,16 @@
   "types": "dist/types/index.d.ts",
   "typings": "dist/types/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "tailwindPlugins",
+    "preset.d.ts",
+    "preset.js"
```

Comment:
Need to include them in the packages files to use them after installing the package.